doc/Ptz.xml

```diff
@@ -932,6 +932,70 @@ Add GeoMove</revremark>
           </varlistentry>
         </variablelist>
       </section>
+      <section>
+        <title>MoveAndStartTracking</title>
+        <para>A device can support the MoveAndStartTracking operation. The pourpose of this action is to send an an atomic command to the device: move the camera to a wanted position and then delegate the PTZ control to the tracking algorithm. With the atomic command the latency is minimized. The move position is optional and can be set in three different modes:</para>
```

Comment:
Add a capability?

Reply:
Adding a "MoveAndStartTrackingSupported" capability to PTZNode would solve also the question raised from the comment below ("how would the client know if the detection and tracking is done in the same device?").
Should I then modify this same pull request to add this capability, or it is better to close this pull request and then open a new pull request that we all can discuss about?